qiskit_sphinx_theme/pytorch_base/layout.html

```diff
@@ -31,14 +31,7 @@
   <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" />
   {%- endfor -%}
 
-  <link rel="modulepreload" href="https://cdn.jsdelivr.net/npm/@qiskit/web-components/experimental-bundled-ui-shell.js">
-  <script type="module" src="https://cdn.jsdelivr.net/npm/@qiskit/web-components/experimental-bundled-ui-shell.js"></script>
-  <style>
-    qiskit-ui-shell:not(:defined) {
-      display: block;
-      margin-bottom: 3.5rem;
-    }
-  </style>
```

Comment:
Not needed anymore! The top nav bar will always be defined when the site renders.